### PR TITLE
Applies minor fixes to RpcConstants.

### PR DESCRIPTION
**Pull request by dinooliva**

### CODE REVIEW

File: core/java/com/google/instrumentation/stats/RpcConstants.java [hunk 1]
```diff
@@ -116,12 +116,12 @@
           MeasurementUnit.create(6, bytes));
 
   // Common histogram bucket boundaries for bytes received/sets DistributionViewDescriptors.
-  private static final List<Double> RPC_BYTES_BUCKET_BOUNDARIES = Arrays.asList(
```
Comment (a-veitch) [inline core/java/com/google/instrumentation/stats/RpcConstants.java]: The List used here is not immutable, so the ALL_CAPS style should not apply. Either change the name, or use Collections.unmodifiableList. (also other similar uses).

File: core/java/com/google/instrumentation/stats/RpcConstants.java [hunk 2]
```diff
@@ -159,7 +159,7 @@
           Arrays.asList(RPC_CLIENT_METHOD));
   public static final DistributionViewDescriptor RPC_CLIENT_RESPONSE_BYTES_VIEW =
       DistributionViewDescriptor.create(
-          "/rpc/client/response_bytes",
+          "rpc client response_bytes",
```
Comment (a-veitch) [inline core/java/com/google/instrumentation/stats/RpcConstants.java]: Have we decided this is the type of naming we want? (we need to have the general view and resource naming discussion at some point).

Comment (dinooliva) [inline core/java/com/google/instrumentation/stats/RpcConstants.java]: No, I think this is still open for discussion - I've made the name here consistent with the other names but they may be changed.

File: core/java/com/google/instrumentation/stats/RpcConstants.java (hunk 1 above)
Comment (dinooliva) [inline core/java/com/google/instrumentation/stats/RpcConstants.java]: They are meant to be constants but that's a reasonable point. I'll change them to be UnmodifiableLists in the next pr.

**Pull request merged**